packages/metamask/src/index.ts

```diff
@@ -63,6 +63,12 @@ export class MetaMask extends Connector {
         })
 
         this.provider.on('disconnect', (error: ProviderRpcError): void => {
+          // 1013 indicates that MetaMask is attempting to reestablish the connection
+          // https://github.com/MetaMask/providers/releases
+          if (error.code === 1013) {
+            console.error('Ignore the error (1013): "MetaMask: Disconnected from chain. Attempting to connect.:"', error);
```

Comment:
If this should be ignored, then it should not be logged as an error.

What about: `console.debug('MetaMask logged connection error 1013: "Try again later"')` (from https://github.com/MetaMask/providers/blob/2068cd5d98561d0dcadde5e13eb1303c9a382d84/src/MetaMaskInpageProvider.js#L416).

Arguably, you could not log anything at all, as the web3-react abstraction takes care of it.

Reply:
Changed to `console.debug`